Singer.API/ClientApp/src/app/modules/careusers/components/care-user-details/care-user-details.component.html

```diff
@@ -18,7 +18,11 @@ <h3 class="section-title">Persoonlijke gegevens</h3>
             />
             <mat-error
                *ngIf="formControlGroup.controls.lastNameFieldControl.invalid"
-               >{{ getRequiredFieldErrorMessage(formControlGroup.controls.lastNameFieldControl) }}</mat-error
+               >{{
+                  getRequiredFieldErrorMessage(
+                     formControlGroup.controls.lastNameFieldControl
+                  )
+               }}</mat-error
```

Comment:
Can the `<mat-error>` opening and closing tags be put on on separate lines, with the content indenting? It would improve readability IMHO

Reply:
@vyruz1986 Should be fixed now.